gpigulevsky/CSS_03/style_menu_auto.css

```diff
@@ -0,0 +1,56 @@
+          ul {
+            list-style-type: none;      /*убираем отступы и маркеры списков */
+            margin: 0;
+            padding: 0;
+            border-bottom: 1px solid black;
+            border-radius: 5px;
+            width: 150px;
+          }
+          li {position: relative;} /* Установка для родительского объекта считать позицию дочернего обьекта относительно него*/
+          ul li {transition: 1s all ease-in-out;}  /*делаем применение стилей постепенно в течение 1 секунды для всех пунктов списка */
+          li ul {
+            display: none;
+            position: absolute;
+            top: 0;
+            left: 149px;
+          }
+          li:hover ul {             /* Скрытие при наведении */
+            display: block;
+          }
+          li a {
+            display: block;
+            border: 1px solid black;
+            border-bottom: none;
+            border-radius: 5px;
+            padding: 3px;
+            background-color: gray;
+            color: white;
+            text-decoration: none;
+            transition: 1s all ease-in-out;
+          }
+          li a:hover {
+            background-color: white;
+            color: gray;
+          }
+          @media all and (min-width: 920px) {
+                      ul {
+                          width: auto;
+                          border-bottom: none;
+                      }
+                      li {                                    /* расположение списка по горизонтали */
+                          display: inline-block;
```

Comment:
why have you used two? Should be enough just one, or `float` or `display`

Reply:
without float, elements of menu have intervals

Reply:
> why have you used two? Should be enough just one, or `float` or `display`

https://files.slack.com/files-pri/TC8AF2ZKL-FQBTB4Q68/____________.png